docs/UsersGuide/source/RunSRW.rst

```diff
@@ -827,18 +831,24 @@ In addition to the baseline tasks described in :numref:`Table %s <WorkflowTasksT
    | **Workflow Task**     | **Task Description**                                       |
    +=======================+============================================================+
    | GET_OBS_CCPA          | Retrieves and organizes hourly :term:`CCPA` data from NOAA |
```

Comment:
@willmayfield @gspetro This is not directly related to this PR, but we will have to update the names of the vx and related tasks in this table, e.g. `GET_OBS_CCPA` becomes `get_obs_ccpa`, etc.  I think I'll do that as part of Issue #630.

Reply:
@gsketefian I'm actually working on updates now! The code is in my text/ug-updates branch.
RTD version here: https://srw-ug.readthedocs.io/en/text-ug-updates/BuildingRunningTesting/RunSRW.html#vxworkflowtaskstable 
I'm generally not touching much of the VX stuff, but since I'm updating the run chapter, I did get update some of the info there.

Reply:
One thing I was trying to decide was whether to name the (meta)tasks based on what is in the YAML config file or based on what the Rocoto/log output is. I went with what is in the YAML because that is the name users may actually add/change/remove in their config file.